buildSrc/src/main/groovy/org/gradle/testretry/build/GradleVersionData.groovy

```diff
@@ -35,4 +42,24 @@ class GradleVersionData {
             .collect { it.key.toString() }
     }
 
+    static int major(GradleVersion gradle) {
+        def version = gradle.version
+        def matcher = MAJOR_AND_MINOR_VERSION_PATTERN.matcher(version)
+        if (matcher.matches()) {
+            return Integer.parseInt(matcher.group(1))
+        } else {
+            throw new IllegalArgumentException("Failed to determine major version for version ${version}")
+        }
+    }
+
+    static int minor(GradleVersion gradle) {
+        def version = gradle.version
```

Comment:
🧐 Maybe `major` and `minor` methods could be collapsed? Or the common part is extracted? Both of them do the matching, just the part of the exception message and the group number are different.

Reply:
Thought about it but first decided against it, since it's such a trivial function. But given that it had a copy/paste error, I changed it. :P

DONE (c201b530)